fenix/app/src/main/java/org/mozilla/fenix/compose/tabstray/TabListItem.kt

```diff
@@ -89,37 +91,43 @@ fun TabListItem(
 
         Column(
             modifier = Modifier
-                .padding(horizontal = 16.dp)
+                .padding(start = 12.dp)
                 .weight(weight = 1f),
         ) {
             Text(
                 text = tab.toDisplayTitle().take(MAX_URI_LENGTH),
+                color = FirefoxTheme.colors.textPrimary,
                 fontSize = 16.sp,
+                letterSpacing = 0.0.sp,
```

Comment:
Is there an existing TextStyle that we could map this to? I have to assume that since our design system doesn't have a text style with 0.0sp letter spacing would indicate that the existing xml style is something that should be updated.

Reply:
It's a sound argument, but I feel hesitant to update the existing wrapping in .xml, it's just too visible of a change (we will go from 0.00 to somewhere around 0.15, which leaves us with less space for text and will cut tab names and links). The task was to bring Compose implementation to the .xml one. If we want to change the existing behaviour (shorten the text in tabs), it's better be a separate task with the UX team taking a look at it.

@MozillaNoah Wonder what you think

Reply:
We already have [one ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=1828684) to ask UX about the text styling of another element in the Tabs Tray, but I'm fine if this ticket gets re-tooled now to instead be confirming with UX what the style should be and saving all of the BZ admin overhead. 

@mavduevskiy, if you choose to defer this work, please file a follow-up and start that conversation with UX and I'll add it to the meta.

Reply:
This is the follow up [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1833493). I have added the unified search meta bug as a blocking for the meta. Who should I ping from the UX team for this kind of topic? I couldn't find anybody mentioned in the UX ticket you gave as an example :(

Reply:
I would post in our engineering + UX slack channel and maybe cc Verdi and Crystal on it. I haven't gotten a UX owner for this yet.